src/main/java/net/william278/velocitab/config/Settings.java

```diff
@@ -164,6 +168,12 @@ public String getFormat(@NotNull String serverGroup) {
                 formats.getOrDefault(serverGroup, "%username%"));
     }
 
+    @NotNull
+    public String getNametag(@NotNull String serverGroup) {
```

Comment:
I think nametags should be optional -- so perhaps having this return an `Optional` like the header/footer would be best here.

Reply:
![image](https://github.com/WiIIiam278/Velocitab/assets/56083016/fcb06b55-411d-4de1-a1e1-b82d6c730fee)
Header and footer methods don't use optional

Reply:
Oh, mb.

Reply:
So do I need to change that?

Reply:
Should be fine as is